app.js

```diff
@@ -106,29 +100,35 @@ function renderImages (){
   allProducts[imageArray[1]].viewed++;
   allProducts[imageArray[2]].viewed++;
 
-  imageArray.pop();
-  imageArray.pop();
-  imageArray.pop();
+  imageArray.shift();
```

Comment:
In terms of functionality, using shift is great, and it's fine to use it here. In terms of organization, it's much better placed in the uniqueImageChecker function.